recipes/gdbn/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "gdbn" %}
+{% set version = "0.1" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "cb3ceac77e89a31632b27d19c2c5d87ccb425c6bf281b8a8d2bdcf83fe8818a6" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
```

could it be ``noarch``?
```suggestion
  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
  noarch: python
```

---

It appears that `noarch: python` did not help. I'm getting the error: 

```
(%PREFIX%) %SRC_DIR%>%PREFIX%\python.exe -m pip install . --no-deps -vv 
Ignoring indexes: https://pypi.org/simple
Created temporary directory: c:\users\vssadm~1\appdata\local\temp\pip-ephem-wheel-cache-xiiw0r
Created temporary directory: c:\users\vssadm~1\appdata\local\temp\pip-req-tracker-miij7h
Created requirements tracker 'c:\\users\\vssadm~1\\appdata\\local\\temp\\pip-req-tracker-miij7h'
Created temporary directory: c:\users\vssadm~1\appdata\local\temp\pip-install-ixww6o
Config variable 'Py_DEBUG' is unset, Python ABI tag may be incorrect
Config variable 'WITH_PYMALLOC' is unset, Python ABI tag may be incorrect
Config variable 'Py_UNICODE_SIZE' is unset, Python ABI tag may be incorrect
Processing c:\miniconda\conda-bld\gdbn_1572808658578\work
  Created temporary directory: c:\users\vssadm~1\appdata\local\temp\pip-req-build-oi8tcv
Cleaning up...
Removed build tracker 'c:\\users\\vssadm~1\\appdata\\local\\temp\\pip-req-tracker-miij7h'
Traceback (most recent call last):
  File ".ci_support\build_all.py", line 134, in <module>
    build_all(args.recipes_dir, args.arch)
  File ".ci_support\build_all.py", line 68, in build_all
    build_folders(recipes_dir, new_comp_folders, arch, channel_urls)
  File ".ci_support\build_all.py", line 124, in build_folders
    conda_build.api.build([recipe], config=get_config(arch, channel_urls))
  File "C:\Miniconda\lib\site-packages\conda_build\api.py", line 209, in build
    notest=notest, need_source_download=need_source_download, variants=variants)
  File "C:\Miniconda\lib\site-packages\conda_build\build.py", line 2344, in build_tree
    notest=notest,
  File "C:\Miniconda\lib\site-packages\conda_build\build.py", line 1459, in build
    windows.build(m, build_file, stats=build_stats, provision_only=provision_only)
  File "C:\Miniconda\lib\site-packages\conda_build\windows.py", line 294, in build
    check_call_env(cmd, cwd=m.config.work_dir, stats=stats, rewrite_stdout_env=rewrite_env)
  File "C:\Miniconda\lib\site-packages\conda_build\utils.py", line 398, in check_call_env
    return _func_defaulting_env_to_os_environ('call', *popenargs, **kwargs)
  File "C:\Miniconda\lib\site-packages\conda_build\utils.py", line 378, in _func_defaulting_env_to_os_environ
    raise subprocess.CalledProcessError(proc.returncode, _args)
subprocess.CalledProcessError: Command '['cmd.exe', '/d', '/c', 'conda_build.bat']' returned non-zero exit status 1.
```